src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import { QueryRenderer, createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/Artsy"
+import { AuctionArtworksRailArtworksQuery } from "v2/__generated__/AuctionArtworksRailArtworksQuery.graphql"
+import { AuctionArtworksRailArtworks_sale } from "v2/__generated__/AuctionArtworksRailArtworks_sale.graphql"
+import { Carousel } from "v2/Components/Carousel"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+// import {
+//   ActionType,
+//   ClickedArtworkGroup,
+//   ContextModule,
+//   OwnerType,
+// } from "@artsy/cohesion"
+import {
+  AUCTION_ARTWORKS_RAIL_HEIGHT,
+  AUCTION_ARTWORKS_IMAGE_HEIGHT,
+} from "./AuctionArtworksRail"
+import { AuctionArtworksRailPlaceholder } from "./AuctionArtworksRailPlaceholder"
+// import { useTracking } from "react-tracking"
+// import { useAnalyticsContext } from "v2/Artsy/Analytics/AnalyticsContext"
+
+export interface AuctionArtworksRailArtworksProps {
+  sale: AuctionArtworksRailArtworks_sale
+}
+
+const AuctionArtworksRailArtworks: React.FC<AuctionArtworksRailArtworksProps> = ({
+  sale,
+}) => {
+  // const tracking = useTracking()
```

Comment:
All of this commented code we'll have to uncomment, once we get some tracking info. So I left it in.